src/lib.rs

```diff
@@ -450,12 +445,11 @@ impl Database {
     /// after loading a large directory with fonts.
     /// Or a specific face from a font.
     pub fn remove_face(&mut self, id: ID) -> bool {
```

Comment:
I don't think we should return `bool` anymore.

And we should rename it to `disable_face`.